S/SLEEF/build_tarballs.jl

```diff
@@ -0,0 +1,70 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+
+name = "SLEEF"
+version = v"3.4.0"
+
+# Collection of sources required to complete build
+sources = [
+    GitSource("https://github.com/shibatch/sleef.git", "8df2bce4596ec6b86f357beebf2e428564f0a4ec"),
+]
+
+# Bash recipe for building across all platforms
+script = raw"""
+cd $WORKSPACE/srcdir/sleef
+mkdir build-native
+cd build-native
+PATH=`echo $PATH | cut -d":" -f 3-` \
+    CC=$HOSTCC \
+    CXX=$HOSTCXX \
+    cmake \
+        -G Ninja \
+        -DBUILD_QUAD=TRUE \
+        ..
+ninja all
+
+cd $WORKSPACE/srcdir/sleef
+mkdir build-cross
+cd build-cross
+cmake \
+    -DCMAKE_INSTALL_PREFIX=$prefix \
+    -DCMAKE_TOOLCHAIN_FILE=${CMAKE_TARGET_TOOLCHAIN} \
+    -DCMAKE_BUILD_TYPE=Release \
+    -G Ninja \
+    -DNATIVE_BUILD_DIR=$WORKSPACE/srcdir/sleef/build-native \
+    -DSLEEF_SHOW_CONFIG=1 \
+    -DBUILD_TESTS=OFF \
+    -DBUILD_DFT=TRUE \
+    -DBUILD_QUAD=TRUE \
+    -DBUILD_INLINE_HEADERS=TRUE \
+    ..
+ninja all
+ninja install
+"""
+
+# These are the platforms we will build for by default, unless further
+# platforms are passed in on the command line
+platforms = supported_platforms()
+
+# The products that we will ensure are always built
+products = [
+    LibraryProduct("libsleef", :libsleef),
+    LibraryProduct("libsleefdft", :libsleefdft),
+    LibraryProduct("libsleefgnuabi", :libsleefgnuabi),
+    LibraryProduct("libsleefquad", :libsleefquad),
+]
+
+# Dependencies that must be installed before this package can be built
+dependencies = [
+    Dependency(PackageSpec(name="CompilerSupportLibraries_jll", uuid="e66e0078-7015-5450-92f7-15fbd957f2ae")),
+    Dependency("MPFR_jll"),
+    Dependency("OpenSSL_jll"),
+    HostBuildDependency("MPFR_jll"),
+    HostBuildDependency("OpenSSL_jll"),
```

Comment:
https://sleef.org/compile.xhtml

>  In order to build SLEEF, you need CMake, which is an open-source and cross-platform building tool. In order to test the library, it is better to have the GNU MPFR Library, Libssl and FFTW. 

Do you really need MPFR and OpenSSL just for building?

Reply:
Probably not...